common/src/lib.rs

```diff
@@ -38,6 +38,8 @@ use storage_queue::StorageQueue;
 
 pub const STORAGE_PROGRAM_PREFIX: &[u8] = b"g::prog::";
 pub const STORAGE_PROGRAM_PAGES_PREFIX: &[u8] = b"g::pages::";
+pub const STORAGE_PROGRAM_STATE_PREFIX: &[u8] = b"g::prog_state::";
```

Comment:
why don't we just store that in the program struct ?

Reply:
no real reason for standalone prefix. Will change